### PR TITLE
updated and improved validation logic

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: src/validators/v1/issues.js [hunk 1]
```diff
@@ -6,5 +6,8 @@ module.exports = {
 			.withMessage('description field is empty')
 			.isString()
 			.withMessage('description is invalid')
+			.not()
+			.matches(/(\b)(on\S+)(\s*)=|javascript:|<(|\/|[^\/>][^>]+|\/[^>][^>]+)>/gi)
```
Comment (rakeshSgr) [inline src/validators/v1/issues.js:10]: @nevil-mathew what all things it will reject here

Comment (rakeshSgr) [reply src/validators/v1/issues.js:10]: is a standerd regex ?

File: src/constants/blacklistConfig.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	update: ['id', 'organization_id', 'created_at', 'updated_at', 'created_by', 'updated_by'],
+}
+
+const profile = {
+	create: [
+		'user_id',
+		'rating',
+		'visibility',
+		'visible_to_organizations',
+		'external_session_visibility',
```
Comment (rakeshSgr) [inline src/constants/blacklistConfig.js:29]: @nevil-mathew password we should not allow  to update, please check in user service also

File: src/validators/v1/issues.js (hunk 1 above)
Comment (nevil-mathew) [reply src/validators/v1/issues.js:10]: @rakeshSgr please refer this
https://www.regextester.com/110397

File: src/constants/blacklistConfig.js (hunk 2 above)
Comment (nevil-mathew) [reply src/constants/blacklistConfig.js:29]: mentoring service is not storing password, it should be rejected in user service

**Pull request merged**